public/iframe2.json

```diff
@@ -9,5 +9,145 @@
+                            {
+                                "type": "accordion",
+                                "props": {
+                                    "parameters": [
+                                        {
+                                            "name": "Spacing horizontal",
+                                            "acceptRejectMode": true,
+                                            "overrides": {
+                                                "displayname": "Horizontal spacing"
+                                            }
+                                        },
+                                        {
+                                            "name": "Spacing vertical",
+                                            "acceptRejectMode": true,
+                                            "overrides": {
+                                                "displayname": "Vertical spacing"
+                                            }
+                                        },
+                                        {
+                                            "name": "Side board",
+                                            "acceptRejectMode": false,
+                                            "overrides": {
+                                                "displayname": "Width of side board"
+                                            }
+                                        },
+                                        {
+                                            "name": "Top board",
+                                            "acceptRejectMode": false,
+                                            "overrides": {
+                                                "displayname": "Width of top board"
+                                            }
+                                        }
+                                    ],
+                                    "exports": [{
+                                        "name": "Download 3D",
+                                        "overrides": {
+                                            "displayname": "Download 3D model",
+                                            "group": {
+                                                "id": "7094b726-d2d0-4d3e-8570-6a4371cb4083",
+                                                "name": "Geometry"
+                                            },
+                                            "order": 100
+                                        }
+                                    }]
+                                }
+                            },
+                            {
+                                "type": "text",
+                                "props": {
+                                    "text": "Here you can change the spacing between boards"
+                                }
+                            }
+                        ]
+                    },
```

Comment:
@mathieu-shapediver parameter and export properties can now be overridden from Grasshopper, this example changes the `displayname` of some parameters and the `group` and `order` of an export. 
Next we should plug this possibility into the parameter and export reference components in Grasshopper.